buildmaster/old_new_porting_map.yml

```diff
@@ -8,63 +8,63 @@
 
 # HERA
 HERACOMBCCEM:
-  new_name: HERA_CC_318GEV_SIGMARED_EM
+  new_name: HERA_CC_318GEV_EM-SIGMARED
   reference_arxiv: '1506.06042'
   reference_hepdata: 10.17182/hepdata.68951.v1/t7
   journal: Eur. Phys. J. C75 (2015) 580
 HERACOMBCCEP:
-  new_name: HERA_CC_318GEV_SIGMARED_EP
+  new_name: HERA_CC_318GEV_EP-SIGMARED
   reference_arxiv: '1506.06042'
   reference_hepdata: 10.17182/hepdata.68951.v1/t6
   journal: Eur. Phys. J. C75 (2015) 580
 HERACOMBNCEM:
-  new_name: HERA_NC_318GEV_SIGMARED_EM
+  new_name: HERA_NC_318GEV_EM-SIGMARED
   reference_arxiv: '1506.06042'
   reference_hepdata: 10.17182/hepdata.68951.v1/t5
   journal: Eur. Phys. J. C75 (2015) 580
 HERACOMBNCEP460:
-  new_name: HERA_NC_225GEV_SIGMARED_EP
+  new_name: HERA_NC_225GEV_EP-SIGMARED
   reference_arxiv: '1506.06042'
   reference_hepdata: 10.17182/hepdata.68951.v1/t4
   journal: Eur. Phys. J. C75 (2015) 580
 HERACOMBNCEP575:
-  new_name: HERA_NC_251GEV_SIGMARED_EP
+  new_name: HERA_NC_251GEV_EP-SIGMARED
   reference_arxiv: '1506.06042'
   reference_hepdata: 10.17182/hepdata.68951.v1/t3
   journal: Eur. Phys. J. C75 (2015) 580
 HERACOMBNCEP820:
-  new_name: HERA_NC_300GEV_SIGMARED_EP
+  new_name: HERA_NC_300GEV_EP-SIGMARED
   reference_arxiv: '1506.06042'
   reference_hepdata: 10.17182/hepdata.68951.v1/t2
   journal: Eur. Phys. J. C75 (2015) 580
 HERACOMBNCEP920:
-  new_name: HERA_NC_318GEV_SIGMARED_EP
+  new_name: HERA_NC_318GEV_EP-SIGMARED
   reference_arxiv: '1506.06042'
   reference_hepdata: 10.17182/hepdata.68951.v1/t1
   journal: Eur. Phys. J. C75 (2015) 580
 HERACOMB_SIGMARED_B:
-  new_name: HERA_NC_318GEV_EAVG_SIGMARED_BOTTOM
+  new_name: HERA_NC_318GEV_EAVG_BOTTOM-SIGMARED
   reference_arxiv: '1804.01019'
   reference_hepdata: ''
   journal: Eur. Phys. J. C78 (2018) 473
 HERACOMB_SIGMARED_C:
-  new_name: HERA_NC_318GEV_EAVG_SIGMARED_CHARM
+  new_name: HERA_NC_318GEV_EAVG_CHARM-SIGMARED
   reference_arxiv: '1804.01019'
   reference_hepdata: ''
   journal: Eur. Phys. J. C78 (2018) 473
 HERAF2CHARM:
-  new_name: HERA_NC_318GEV_EAVG_SIGMARED_F2
+  new_name: HERAF2CHARM_NC_318GEV_EAVG_F2-SIGMARED
```

Comment:
I am not sure about this name... why is there F2CHARM in the experiment name? the naming format is <exp>_<process>_<energy or lack thereof (not fixed)>_<misc_details_like_lumi_etc.>_<obs-name-and-misc-additives>. If its imp to have f2charm in the name, it should be in the misc. section, i.e. it should be like HERA_NC_318GEV_F2CHARM_EAVG_<obs-name> or HERA_NC_318GEV_EAVG_F2CHARM_<obs-name> or even HERA_NC_318GEV_EAVG_CHARM_<obs-name>

Reply:
Ups. This is one of the ones coming from older times (not used in the 4.0 runcard dataset). It has the experiment set as HERAF2CHARM and so I need to either set a special case for it in my script to autofill this or remove it.